Makefile

```diff
@@ -1,5 +1,5 @@
 BIN_NAME="terraform-provider-instaclustr"
-VERSION=v1.5.0
+VERSION=v1.5.1
```

Comment:
[Required, correctness] - since this isn't a bugfix this should be a minor version bump

Reply:
Roger will address cheers